sql-cli/tests/config/default/configuration.yaml

```diff
@@ -0,0 +1,48 @@
+connections:
```

Comment:
Do we need such a long file? Could we have a simpler one?

Reply:
Trimmed the file to have only 2 connections (including one with vars)